packages/altive_lints/README.md

```diff
@@ -118,6 +118,7 @@ CustomScrollView(
 ### avoid_hardcoded_color
 
 Do not use hard-coded Color.
+However, `Colors.transparent` can be used.
```

Comment:
As mentioned in [this comment](https://github.com/altive/altive_lints/pull/63#pullrequestreview-2350833726), I thought it would be kind to add an explanation, so I added it!